packages/apps/kadena-docs/src/components/Layout/components/Aside/AsideStyles.ts

```diff
@@ -0,0 +1,226 @@
+import { styled, StyledComponent } from '@kadena/react-components';
+
+export const BaseBackground: StyledComponent<
+  'div',
+  {
+    isOpen?: boolean | 'true' | 'false' | undefined;
+  }
+> = styled('div', {
+  position: 'absolute',
+  zIndex: 2,
+  pointerEvents: 'none',
+  width: '100vw',
+  height: '100vh',
+
+  transform: 'translateX(100vw)',
+
+  '@md': {
+    position: 'absolute',
+    transform: 'translateX(0)',
+  },
+
+  variants: {
+    isOpen: {
+      true: {},
+      false: {},
+    },
+  },
+
+  '&::after': {
+    content: '',
+    position: 'absolute',
+    inset: 0,
+    backgroundColor: '$backgroundOverlayColor',
+    zIndex: 1,
+  },
+});
+
+export const CodeBackground: StyledComponent<typeof BaseBackground> = styled(
+  BaseBackground,
+  {
+    '--shadowWidth': '$sizes$20',
```

Comment:
I recommend using `$$shadowWidth` for locally scoped tokens 
https://stitches.dev/docs/styling#locally-scoped-tokens